### ISSUE TITLE
Normalize.to_normalized_graphql ignores annotations

### ISSUE THREAD

**Issue by barolick** [rmosolgo/graphql-ruby#4703]
**Describe the bug**

When attempting to add operation batch with two identical operations that are differentiated only by annotations, only one operation is added. This seems to be due to the fact that the `Normalize.to_normalized_graphql` function generates the same string for both operations.

**Versions**

`graphql` version: 1.12.19
`graphql-pro` version: 1.24.14
`rails` (or other framework): 6.1.7.6

**Expected behavior**

When calling add_operation_batch with identical operations differentiated by annotations, both should be inserted into the operation store.

**Actual behavior**

Only the first instance of the operation is being inserted

**Comment by barolick** [comment]
For example, these two operations would be considered duplicates
```
{
  "014c9ef9d8115d5a0ffe212703fd7f45": "query ProfileModalQuery {\n  viewer {\n id\n  } \n }\n",
  "fe4b7da9e9efe8b0361b42e7772268a6": "query ProfileModalQuery @codeOwner(owner: \"team-oxygen\") {\n  viewer {\n id\n  } \n }\n"
}
```

**Comment by rmosolgo** [comment]
Hey, thanks for the detailed report! I just released GraphQL-Pro v1.24.15 which should address this issue. Could you give that new version a try and let me know if you encounter any trouble with it? 

I found that directives were not addressed at all on Operation definitions (or on Fragment definitions, which I fixed, or Variable definitions, which aren't supported by the parser yet: #4706). So I added directive handling there! It also used to sort directive definitions, which was the wrong behavior, so I removed that.
